.github/workflows/features.yml

```diff
@@ -50,3 +50,4 @@ jobs:
           pytest tests/test_functions_ids.py
           pytest tests/test_function.py
           pytest tests/test_source_mapping.py
+          pytest tests/test_detectors_mocked_contract.py
```

Comment:
I think we should use the same testing strategy for all the detectors, otherwise it will be difficult to maintain. We can use `fopwCDKKK()` to simulate a collision with `DOMAIN_SEPARATOR()`

Reply:
I hadn't found a collision on 4bytes, which is why I mocked it and mutated the property. Thanks for finding this as I can test it normally.